client/my-sites/current-site/stale-cart-items-notice.js

```diff
@@ -40,7 +40,7 @@ class StaleCartItemsNotice extends React.Component {
 				id: staleCartItemNoticeId,
 				isPersistent: false,
 				duration: 10000,
-				button: this.props.translate( 'Complete your purchase' ),
+				button: this.props.translate( 'View your cart' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'View your cart:' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👍